compose/ui/ui/src/desktopMain/kotlin/androidx/compose/ui/awt/SwingPanel.desktop.kt

```diff
@@ -144,8 +145,10 @@ public fun <T : Component> SwingPanel(
         }
     }
 
-    SideEffect {
+    LaunchedEffect(background) {
```

Comment:
Not sure, that it is the same. It runs coroutine that can be heavy, and it can be dispatched differently (after SideEffect, for example, or after all another `LaunchedEffect`'s).

Do Jetpack Compose have recomendations for such cases? Usually I saw that it uses `SideEffect` internally for this. if we need performance, `componentInfo.container.background` usually compares the new value with the old value.